### PR TITLE
Fetch APIs plists

### PR DESCRIPTION
**Pull request by viikufa**
* added fetching from remote keystore repository apiKeys
* added check apiKeys exists

### CODE REVIEW

File: station.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4355,6 +4360,66 @@
...
+				"$(SRCROOT)/station/Resources/Plists/GoogleService-Info.plist",
+			);
+			name = "Fetch secrets";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "BUILD_APP_DIR=${BUILT_PRODUCTS_DIR}/${FULL_PRODUCT_NAME}\ngit clone git@github.com:ruuvi/com.ruuvi.station.ios.keystore.git\nif [ $? -eq 0 ]; then\n    /bin/cp -rf com.ruuvi.station.ios.keystore/GoogleService-Info.plist \"$BUILD_APP_DIR/GoogleService-Info.plist\"\n    /bin/cp -rf com.ruuvi.station.ios.keystore/Networking.plist \"$BUILD_APP_DIR/Networking.plist\"\n    rm -rf com.ruuvi.station.ios.keystore\nelse\n    if grep -q \"{ set your API key here }\" $SCRIPT_INPUT_FILE_0; then\n        echo \"warning: Need setup OpenWeather ApiKey in /Classes/Networking/Assembly/Networking.plist\"\n    fi\n    if grep -q \"1:925543306936:ios:84f5fda343c52e7671c64d\" $SCRIPT_INPUT_FILE_1; then\n        echo \"warning: If you want use GoogleServices need replace file Resources/Plists/GoogleService-Info.plist\"\n    fi\nfi\n";
```
Comment (rinat-enikeev) [inline station.xcodeproj/project.pbxproj]: "warning: Need setup OpenWeather ApiKey in /Classes/Networking/Assembly/Networking.plist\"

-> 

"Missing OpenWeatherMap API key. In order to make Virtual Sensors work please obtain API key on https://openweathermap.org and put into station/Classes/Networking/Assembly/Networking.plist"

Comment (rinat-enikeev) [inline station.xcodeproj/project.pbxproj]: "warning: If you want use GoogleServices need replace file Resources/Plists/GoogleService-Info.plist\" 

-> 

"Demo GoogleServices credentials. If you want to use your own GoogleServices credentials, please replace the station/Resources/Plists/GoogleService-Info.plist file"

**Comment by DG12** [comment]
It would be really helpful if there were comments like what it does, what was wrong and how these changes make it better. 
Will even you remember what this is about a month from now? a year?

**Comment by rinat-enikeev** [comment]
@DG12 this is internal design. We are automating the private keys setup in the Ruuvi Station.

**Pull request merged**